app/noon_forecasts.py

```diff
@@ -0,0 +1,90 @@
+""" This module is used to fetch noon forecasts for weather stations from
+the noon_forecasts table in our database.
+"""
+import logging
+import datetime
+import math
+import pytz
+from sqlalchemy import desc
+from app.schemas import NoonForecast, NoonForecastResponse, NoonForecastValue, StationCodeList
+import app.db.database
+from app.db.models import NoonForecasts
+
+#pylint: disable=invalid-name
+
+LOGGER = logging.getLogger(__name__)
+
+
+class StationNotFoundException(Exception):
+    """ Custom exception for when a station cannot be found """
+
+
+def parse_table_records_to_noon_forecast_response(data: [NoonForecasts]):
+    """ Given a list of table records from the database, parse each record
+    (which is a NoonForecasts object) and structure it as a NoonForecast
+    object, then return the list of NoonForecast objects as a NoonForecastResponse
+    """
+    noon_forecasts = {}
+    # offset is used to convert Pacific Daylight time (noon) to UTC
+    UTC_OFFSET = +7
```

Comment:
Please remove - the database should now be storing the timestamp with a timezone, so you don't have to do anything here - the offset should just be there.